hack/ci/e2e-k8s.sh

```diff
@@ -262,7 +272,7 @@ run_tests() {
   # interrupt
   ./hack/ginkgo-e2e.sh \
     '--provider=skeleton' "--num-nodes=${NUM_NODES}" \
-    "--ginkgo.focus=${FOCUS}" "--ginkgo.skip=${SKIP}" \
+    "--ginkgo.focus=${FOCUS}" "--ginkgo.skip=${SKIP}" "--ginkgo.label-filter=${LABEL_FILTER}" \
```

Comment:
should we avoid to add the flag ` "--ginkgo.label-filter=${LABEL_FILTER}" \` if LABEL_FILTER is empty?

Reply:
If it's empty, then it doesn't make a difference. I didn't want to make the shell code more complicated by adding further if checks.

Reply:
fair